### ISSUE TITLE
.../lib/git-hub.d/bash+.bash: line 1: ../../ext/bashplus/lib/bash+.bash: No such file or directory

### ISSUE THREAD

**Issue by OleksiyRudenko** [ingydotnet/git-hub#254]
Hi!

First of all, your [git-subrepo](https://github.com/ingydotnet/git-subrepo) is just great 
and works like a charm.

I have installed `git-hub` under **Windows Git Bash** by
```
git clone https://github.com/ingydotnet/git-hub ~/bin/git-hub
echo 'source ~/bin/git-hub/.rc' >> ~/.bashrc
```

Now any `git hub` command reports ```/c/Users/oleksiy/bin/git-hub/lib/git-hub.d/bash+.bash: line 1: ../../ext/bashplus/lib/bash+.bash: No such file or directory```

My idea was that default permissions under `~/bin/` might be insufficient, 
so I placed `git-hub` in a place similar to where `git-subrepo` is installed at (and `git-subrepo` works fine)
and changed `source` instruction in `.bashrc` as appropriate.

Now `.bashrc` looks as follows:
```
source "/d/Program Files.portable/git.scripts/git-subrepo/.rc"
source "/d/Program Files.portable/git.scripts/git-hub/.rc"
```

Now **Windows Git Bash** at launch reports:
```
bash: cd: $'/d\nFiles.portable/git.scripts/git-hub': No such file or directory
bash: /c/Users/oleksiy/share/enable-completion.sh: No such file or directory
```
And `git hub <anything>` reports `git: 'hub' is not a git command. See 'git --help'.`

Will appreciate any suggestions on the above.

**Comment by mreedmdev** [comment]
> Hi!
> 
> First of all, your [git-subrepo](https://github.com/ingydotnet/git-subrepo) is just great and works like a charm.
> 
> I have installed `git-hub` under **Windows Git Bash** by
> 
> ```
> git clone https://github.com/ingydotnet/git-hub ~/bin/git-hub
> echo 'source ~/bin/git-hub/.rc' >> ~/.bashrc
> ```
> 
> Now any `git hub` command reports `/c/Users/oleksiy/bin/git-hub/lib/git-hub.d/bash+.bash: line 1: ../../ext/bashplus/lib/bash+.bash: No such file or directory`
> 
> My idea was that default permissions under `~/bin/` might be insufficient, so I placed `git-hub` in a place similar to where `git-subrepo` is installed at (and `git-subrepo` works fine) and changed `source` instruction in `.bashrc` as appropriate.
> 
> Now `.bashrc` looks as follows:
> 
> ```
> source "/d/Program Files.portable/git.scripts/git-subrepo/.rc"
> source "/d/Program Files.portable/git.scripts/git-hub/.rc"
> ```
> 
> Now **Windows Git Bash** at launch reports:
> 
> ```
> bash: cd: $'/d\nFiles.portable/git.scripts/git-hub': No such file or directory
> bash: /c/Users/oleksiy/share/enable-completion.sh: No such file or directory
> ```
> 
> And `git hub <anything>` reports `git: 'hub' is not a git command. See 'git --help'.`
> 
> Will appreciate any suggestions on the above.



> Hi!
> 
> First of all, your [git-subrepo](https://github.com/ingydotnet/git-subrepo) is just great and works like a charm.
> 
> I have installed `git-hub` under **Windows Git Bash** by
> 
> ```
> git clone https://github.com/ingydotnet/git-hub ~/bin/git-hub
> echo 'source ~/bin/git-hub/.rc' >> ~/.bashrc
> ```
> 
> Now any `git hub` command reports `/c/Users/oleksiy/bin/git-hub/lib/git-hub.d/bash+.bash: line 1: ../../ext/bashplus/lib/bash+.bash: No such file or directory`
> 
> My idea was that default permissions under `~/bin/` might be insufficient, so I placed `git-hub` in a place similar to where `git-subrepo` is installed at (and `git-subrepo` works fine) and changed `source` instruction in `.bashrc` as appropriate.
> 
> Now `.bashrc` looks as follows:
> 
> ```
> source "/d/Program Files.portable/git.scripts/git-subrepo/.rc"
> source "/d/Program Files.portable/git.scripts/git-hub/.rc"
> ```
> 
> Now **Windows Git Bash** at launch reports:
> 
> ```
> bash: cd: $'/d\nFiles.portable/git.scripts/git-hub': No such file or directory
> bash: /c/Users/oleksiy/share/enable-completion.sh: No such file or directory
> ```
> 
> And `git hub <anything>` reports `git: 'hub' is not a git command. See 'git --help'.`
> 
> Will appreciate any suggestions on the above.

DId make any progress on this issue?

**Comment by OleksiyRudenko** [comment]
@mreedmdev no, i didn't make any progress on this. Resorted to https://github.com/github/hub instead

**Comment by vhp1360** [comment]
I faced this too.
I put git-subrepo in `/usr/local/share`  and put `. /usr/local/share/git-subrepo/.r` in `.bashrc`
it works one time !!!!.
